README.md

```diff
@@ -58,7 +58,7 @@ We hope to accomplish these goals by:
 | [<img src="https://avatars1.githubusercontent.com/u/14366048?v=4" width="100px;"/><br /><sub><b>Evgenii Bezpalko</b></sub>](https://github.com/Degron)  | [<img src="https://avatars0.githubusercontent.com/u/6467349?v=4" width="100px;"/><br /><sub><b>Jennifer Cen</b></sub>](https://github.com/cenjennifer) | [<img src="https://avatars2.githubusercontent.com/u/16601510?v=4" width="100px;"/><br /><sub><b>Jonathan Schwarz</b></sub>](https://github.com/jes708) |    [<img src="https://avatars2.githubusercontent.com/u/2086094?v=4" width="100px;"/><br /><sub><b>lauradhahn</b></sub>](https://github.com/lauradhahn)     |   [<img src="https://avatars3.githubusercontent.com/u/5679105?v=4" width="100px;"/><br /><sub><b>Maedeh Safari</b></sub>](https://github.com/msafari)    |               [<img src="https://avatars1.githubusercontent.com/u/1974993?v=4" width="100px;"/><br /><sub><b>Malek Hakim</b></sub>](http://malekhakim.me)               | [<img src="https://avatars1.githubusercontent.com/u/1365995?v=4" width="100px;"/><br /><sub><b>Rina Krevat</b></sub>](https://http://rinakrevat.com/) |
 | [<img src="https://avatars0.githubusercontent.com/u/5322552?v=4" width="100px;"/><br /><sub><b>ronhenderson</b></sub>](https://github.com/ronhenderson) | [<img src="https://avatars3.githubusercontent.com/u/8871080?v=4" width="100px;"/><br /><sub><b>sagar kaurav</b></sub>](https://github.com/sagarkaurav) | [<img src="https://avatars3.githubusercontent.com/u/15033101?v=4" width="100px;"/><br /><sub><b>sivdahlberg</b></sub>](https://github.com/sivdahlberg) |    [<img src="https://avatars2.githubusercontent.com/u/2132853?v=4" width="100px;"/><br /><sub><b>Steve Dalonzo</b></sub>](https://github.com/sdalonzo)    | [<img src="https://avatars2.githubusercontent.com/u/2953256?v=4" width="100px;"/><br /><sub><b>Stephen Sulik</b></sub>](https://github.com/stephensulik) | [<img src="https://avatars1.githubusercontent.com/u/16071129?v=4" width="100px;"/><br /><sub><b>Torng-Lin (Taffy) Chen</b></sub>](https://github.com/tornglintaffychen) |      [<img src="https://avatars1.githubusercontent.com/u/45636494?v=4" width="100px;"/><br /><sub><b>sqott</b></sub>](https://github.com/sqott)       |
-|   [<img src="https://avatars1.githubusercontent.com/u/862774?v=4" width="100px;"/><br /><sub><b>Tobias Lundgren</b></sub>](https://lundgren.design/)    |   [<img src="https://avatars1.githubusercontent.com/u/3496298?v=4" width="100px;"/><br /><sub><b>zerodom30</b></sub>](https://github.com/zerodom30)    |
+|   [<img src="https://avatars1.githubusercontent.com/u/862774?v=4" width="100px;"/><br /><sub><b>Tobias Lundgren</b></sub>](https://lundgren.design/)    |   [<img src="https://avatars1.githubusercontent.com/u/3496298?v=4" width="100px;"/><br /><sub><b>zerodom30</b></sub>](https://github.com/zerodom30)    |  [<img src="https://avatars1.githubusercontent.com/u/3157593?v=4" width="100px;"/><br /><sub><b>Kyle Pollock</b></sub>](https://github.com/unixchad)   |
```

Comment:
@unixchad, FYI the Collaborator thumbnails are actually alphabetically sorted, if you want your thumbnail in that sort.